cdap-ui/app/cdap/components/ConfigurationGroup/utilities/DynamicPluginFilters.ts

```diff
@@ -42,28 +42,25 @@ export interface IFilteredConfigurationGroup extends IConfigurationGroup {
 
 /**
  * CDAP UI provides some simple operations like @{ICustomOperators}
- * This is to convert those simple operators to jexl expressions
+ * This is to evaluate the condition object for simpler cases.
  */
-function getExpressionFromConditionObj(filter) {
-  const { property, value, operator } = filter.condition;
-  let expression;
+function evaluateConditionObj(filter: IPropertyFilter, propertyValues: IPropertyValues) {
+  const { property, operator } = filter.condition;
+  const { value } = filter.condition;
+  /* tslint:disable */
   switch (operator) {
     case CustomOperator.EQUALTO:
-      expression = `${property} == ${value}`;
-      break;
+      return propertyValues[property] == value;
```

Comment:
I'm wondering if we should do `value.toString()`? This is to address in case the user puts `value: 10` or `value: true` instead of `value: "10"`?

Also, should this be `===` and `!==`?

Reply:
Have fixed case to handle different types of value.